enigbe/PrinterConnect
# create client avatar

### What does this PR do?
- allow client RUD avatars
### Description of task to be completed?
- Upload an avatar: `PUT '/client/avatar/upload'` to replace existing or default avatar

- Read avatar: `GET '/client/avatar'`

- Delete avatar: `DELETE '/client/avatar'`

### How should this be manually tested?
With postman:
- Upload avatar: Send a `PUT` request to `/client/avatar/upload`. The headers for this request should be `{"Content-Type": "multipart/form-data", "Authorization": "Bearer {{access_token}}"}`, where `{{access_token}}` is your valid JWT from signing in. The body of this request should contain a key-value pair of `image` and the image/avatar you want uploaded.

- Read/view avatar: Send a `GET` request `/client/avatar` 

- Delete avatar: Send a `DELETE` request to `/client/avatar` with `{"Authorization": "Bearer {{access_token}}"}` where `{{access_token}}` is a fresh JWT
